### ISSUE TITLE
VSCode >= 1.13 greys out audio controls in previewHTML?

### ISSUE THREAD

**Issue by anuj-dev** [Microsoft/vscode#32540]
- VSCode Version: >= 1.13
- OS Version: OSX, Ubuntu

Steps to Reproduce:

1. Use any extension that will allow you to preview HTML (e.g. Jupyter or HTML Preview) 
2. Render:
`<audio controls autoplay src="http://media.w3.org/2010/07/bunny/04-Death_Becomes_Fur.oga"></audio>`
3. Observe that the controls are greyed out and the audio does not play

This is broken from 1.13 on, before that previewing audio just worked.

Reproduces without extensions: unknown

**Comment by DickvdBrink** [comment]
![image](https://user-images.githubusercontent.com/765657/29317943-76c3db6a-81ce-11e7-9b34-4b6968e911ac.png)

I get the above when opening a markdown as html; probably due to [this](https://github.com/Microsoft/vscode/blob/f980319f5c4bf3f0878c5fcc68a3aa2795119c57/extensions/markdown/src/previewContentProvider.ts#L278).
The markdown code has a few options to disable the security options; maybe something like that has to be implemented in the extension to?

**Comment by anuj-dev** [comment]
I don't think it's CSP related, the following renders the image ok:

<audio controls autoplay src="http://media.w3.org/2010/07/bunny/04-Death_Becomes_Fur.oga"></audio>
<img src='http://www.w3.org/html/logo/downloads/HTML5_Logo_128.png'/>

**Comment by anuj-dev** [comment]
Oh, from the same version on, it seems that video is also not working (still using the HTML preview to check the rendering)? 
```
<audio controls autoplay src="http://media.w3.org/2010/07/bunny/04-Death_Becomes_Fur.oga"></audio>
<video controls autoplay>
    <source src="http://media.w3.org/2010/05/sintel/trailer.mp4"
         type='video/mp4; codecs="avc1, mp4a"'>
    <source src="http://media.w3.org/2010/05/sintel/trailer.ogv"
         type='video/ogg; codecs="theora, vorbis"'>
</video>
<img src='http://www.w3.org/html/logo/downloads/HTML5_Logo_128.png'/>

```

**Comment by mjbvz** [comment]
Yes, the markdown preview would block http video by default but the previewHtml content should not have any Content Security policy set. Looking at the network requests for the webview, I do see the video file being requested successfully but there seems to be some error with the playback inside the webview. I'll investigate further

**Comment by mjbvz** [comment]
This appears to be fixed in VS Code 1.20 insiders builds:

![screen shot 2017-12-19 at 3 21 38 pm](https://user-images.githubusercontent.com/12821956/34183576-58198498-e4d0-11e7-9463-5ff718b5544d.png)


Let me know if you are still seeing this in html previews

**Comment by mjbvz** [comment]
Actually, I just learned that we strip out ffmpeg and other media support from our release builds. This is by-design and something we will not change, but it does mean that many types of media playback will never be supported in webviews inside vscode
